tfm/targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_064S2_4343W/device/COMPONENT_CM4/device_cfg.h

```diff
@@ -1,41 +0,0 @@
-/*
- * Copyright (c) 2017-2018 Arm Limited
- * Copyright (c) 2020, Cypress Semiconductor Corporation. All rights reserved.
```

Comment:
This file needs to go back to `CY8CKIT_064B0S2_4343W`

Reply:
OK